smithy-rules-engine/src/test/java/software/amazon/smithy/rulesengine/language/IntegrationTest.java

```diff
@@ -131,6 +137,15 @@ void checkInvalidRules(ValidationTestCase validationTestCase) throws IOException
                 error.toString().replaceAll("\\s+", " ").trim());
     }
 
+    @ParameterizedTest
+    @MethodSource("invalidStandaloneValidationTestCases")
+    void checkInvalidStandaloneValidationRules(ValidationTestCase validationTestCase) {
+        EndpointRuleSet ruleset = EndpointRuleSet.fromNode(validationTestCase.contents());
+        List<ValidationError> errors = StandaloneRulesetValidator.validate(ruleset, null)
+            .collect(Collectors.toList());
+        assertFalse(errors.isEmpty());
```

Comment:
Can this assert that the errors list contains the event we care about from the validator?